bigbluebutton-web/grails-app/conf/spring/doc-conversion.xml

```diff
@@ -25,7 +25,7 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
 
     <bean id="documentConversionService" class="org.bigbluebutton.presentation.DocumentConversionServiceImp">
         <property name="bbbWebApiGWApp" ref="bbbWebApiGWApp"/>
-        <property name="officeToPdfConversionService" ref="officeToPdfConversionService"/>
+        <property name="officeToPdfConversionService" ref="officeToPdfConversion"/>

```

Comment:
Please keep the name `officeToPdfConversionService` ( as we removed the old one )